modules/hooks/chat.js

```diff
@@ -297,6 +297,22 @@ export default function() {
     }
   });
 
+ 
+/**
+ * Should remove the socket request message, however doesn't seem to work as expected. 
+ */
+
+  Hooks.on("renderChatMessage", async (app, html, msg) => {
+    const elems = html.find(".message-content > .requestmessage").parents('li');
+      if (elems.length > 0) {
+        if (!CONFIG.debug.wfrp4e) {
+          elems.remove();
+        }
+        WFRP_Utility.log(msg.message.content, true);
+    }
+  });
+
+
```

Comment:
If it doesn't work let's not include it, if I feel inclined I can research or experiment on a better way to do it